config/module.config.php

```diff
@@ -16,14 +16,14 @@
  * and is licensed under the MIT license.
  */
 
-return array(
-    'service_manager' => array(
-        'factories' => array(
+return [
+    'service_manager' => [
+        'factories' => [
             'ZfrCors\Mvc\CorsRequestListener' => 'ZfrCors\Factory\CorsRequestListenerFactory',
```

Comment:
Absolutely; we did that throughout Apigility; let me know if you'd like me
to make this change.

On Aug 16, 2016 4:50 PM, "Michaël Gallego" <notifications@github.com> wrote:

> In config/module.config.php
> <https://github.com/zf-fr/zfr-cors/pull/38#discussion_r75027395>:
>
> > @@ -16,14 +16,14 @@
> >   * and is licensed under the MIT license.
> >   */
> >
> > -return array(
> > -    'service_manager' => array(
> > -        'factories' => array(
> > +return [
> > +    'service_manager' => [
> > +        'factories' => [
> >              'ZfrCors\Mvc\CorsRequestListener' => 'ZfrCors\Factory\CorsRequestListenerFactory',
>
> Could we use ::class ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zf-fr/zfr-cors/pull/38/files/a9bbdf9c81cfe3c14553208bb60d4558683168fa#r75027395>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABlV1heQ5Sla-6cwy7uDkS7WE6j2Qj2ks5qgjCNgaJpZM4JloTx>
> .
>